pipeline/pipeline.go

```diff
@@ -127,6 +128,13 @@ func (r *Runtime) exec(proc *backend.Step) error {
 		}
 	}
 
+	// TODO: using DRONE_ will be deprecated with 0.15.0. remove fallback with following release
```

Comment:
@6543 This is now generating the `DRONE_` environment variables based on the `CI_` versions by simply replacing the prefixes. This way seemed to be much easier to maintain IMO.

Reply:
I placed it directly before executing the pipeline step, so every environment variable will be caught (so it works for cli this way as well).

Reply:
oh! - ok I'll review tomorow